man/systemd.network.xml

```diff
@@ -202,6 +202,8 @@
           <para>The maximum transmission unit in bytes to set for the
           device. The usual suffixes K, M, G, are supported and are
           understood to the base of 1024.</para>
+          <para>Note if IPv6 is enabled and MTU is less than IPv6 Minimum bytes that is
```

Comment:
Maybe:

`Note that if IPv6 is enabled on the interface, and the MTU is chosen below 1280 (the minimum MTU for IPv6) it will automatically be increased to this value.` or so